pom.xml

```diff
@@ -240,7 +240,7 @@
                         -Dhazelcast.test.use.network=false
                     </argLine>
                     <includes>
-                        <include>**/**.java</include>
+                        <include>com/hazelcast/replicatedmap/**.java</include>
```

Comment:
Yes it will be reverted, currently I'm just running the repmap tests in my own jenkins build and running all takes too much time :)